packages/headless/src/insight.index.ts

```diff
@@ -329,6 +329,7 @@ export type {NumericFacetValue} from './features/facets/range-facets/numeric-fac
 export * from './features/facets/range-facets/numeric-facet-set/numeric-facet-actions-loader.js';
 export * from './features/fields/fields-actions-loader.js';
 export * from './features/folding/insight-folding-actions-loader.js';
+export * from './features/generated-answer/generated-answer-actions-loader.js';
```

Comment:
From the PR description:

> The new generateAnswer action is now exposed in both the Headless Insight and Headless Search bundles.

I only see it exposed in the insight bundle. Is this as desired?